src/gui/N2Go_Gui.php

```diff
@@ -136,8 +136,6 @@ public function adminOptions()
         if (strlen($formUniqueCode) > 0 && !isset($forms[$formUniqueCode])) {
             $this->save_option('n2go_formUniqueCode', null);
             $formUniqueCode = null;
-        } else {
-            $form = $forms[$formUniqueCode];
```

Comment:
why is this removed. won't this break anything?

Reply:
`$forms` is an array and will be false if the authKey is not set in the db. 
removing this will fore sure break something
check  https://github.com/newsletter2go/newsletter2go-wordpress-plugin/blob/82fbcef114de09de9b0c87107dfcc943be83494f/src/gui/N2Go_Gui.php#L171

Reply:
The thing is that it does not break anything, just emits warning. Note that `$form` is not used anywhere anyways. It's ok if `$forms` is false, as we care only when it's an array, and even then we don't do anything with `$form`

Reply:
you are right. `$form` is not used anywhere